HousingRepairsSchedulingApi/serverless.yml

```diff
@@ -26,6 +26,7 @@ functions:
     name: ${self:service}-${self:provider.stage}
     handler: HousingRepairsSchedulingApi::HousingRepairsSchedulingApi.LambdaEntryPoint::FunctionHandlerAsync
     role: lambdaExecutionRole
+    timeout: 30
```

Comment:
why does this need to be set to 30 seconds?

Reply:
This is to allow multiple requests to be sent to DRS when searching for available appointments ([Timing reference from Lincoln](https://lbhackney-it.github.io/housing-repairs-online/spikes/scheduled-repairs#appendix-a)).